ui.frontend/src/view/FormTabs.js

```diff
@@ -116,6 +116,11 @@ class FormTabs extends FormPanel {
                         tabs[i].setAttribute(Constants.TABINDEX, "0");
                         tabs[i].setAttribute(Constants.ARIA_CURRENT, "true");
                     } else {
+                         // If the panel/tab was previously active, mark it as stepped
+                         if (tabpanels[i].classList.contains(this.#_selectors.active.tabpanel)) {
```

Comment:
Can you validate the following use-cases,
1. If tabs are dynamically added/removed (via repeatable settings), this indexing might break. Please add test cases for this use-case

for all layout types

Reply:
since it is being directly added to HTML, and is not something which is repopulated, so this might work.
Whenever we introduce lazy loading, at that time this will break.
Nevertheless, we should have a repeatable test cases.